app/controllers/projects/protected_branches_controller.rb

```diff
@@ -20,7 +20,9 @@ def update
 
     if protected_branch &&
        protected_branch.update_attributes(
-        developers_can_push: params[:developers_can_push]
+        developers_can_push: params[:developers_can_push],
+	developers_can_merge: params[:developers_can_merge],
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Tab detected.